openedx/features/edly/admin.py

```diff
@@ -0,0 +1,22 @@
+"""
+Django Admin pages for Edly.
+"""
+
+from django.contrib import admin
+
+from .models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationAdmin(admin.ModelAdmin):
+    list_display = ['name', 'slug']
+
+
+admin.site.register(EdlyOrganization, EdlyOrganizationAdmin)
+
+
+class EdlySubOrganizationAdmin(admin.ModelAdmin):
+    search_fields = ['lms_site__domain', 'lms_site__name', 'studio_site__domain', 'studio_site__name']
+    list_display = ['edx_organization', 'edly_organization']
```

Comment:
We need to show
`edly organization name`, `edly organization slug`, `edly sub organization name`, `edly sub organization short_name`
Also show created, updated timestamps

Reply:
I have used the same format of display as used in edx-organizations app in Django Admin:

<img width="1676" alt="Screenshot 2020-05-04 at 2 59 12 PM" src="https://user-images.githubusercontent.com/17013371/80954988-dc526200-8e17-11ea-992a-768372bf641a.png">

Reply:
Please update admin according to my comment.

Reply:
Done